platforms/aws-s3.md

```diff
@@ -2,6 +2,10 @@
 
 This defines the Amazon Web Services (AWS) S3 interface.
 
-- `platform`: `https://{bucket}.s3.{region}.amazonaws.com`
-- `bucket`: The bucket name
-- `region`: One of the S3 regions (lowercase)
+- `platform`: `https://{bucket}.s3.{region}.amazonaws.com`,
+  which is the endpoint URL after replacing all variables in the URL.
+- `bucket`: The bucket name.
+- `region`: One of the S3 regions (lowercase).
+
+**Note:** If `s3` exists in `auth:refs`, you should use sign requests,
```

Comment:
I don't get here why it is written that: "e.g. using the AWS CLI parameter `--no-sign-request`"

`--no-sign-request` is used when no sign request is needed as name suggests i.e. when bucket is public.

Reply:
Indeed, we'll clarify this here and in the auth extension: https://github.com/stac-extensions/authentication/issues/37

Reply:
```suggestion
**Note:** If the `s3` authentication scheme (i.e. "Simple S3 authentication") is referred to through `auth:refs`, you should disable signing requests,
```